lms/views/basic_lti_launch.py

```diff
@@ -42,6 +42,10 @@ def __init__(self, context, request):
         self.context.js_config.enable_lti_launch_mode()
         self.context.js_config.maybe_set_focused_user()
 
+        request.find_service(name="application_instance").get().update_lms_data(
```

Comment:
I'm not a fan of having "side effects" on a constructor but here we do some similar things and this is not an object we are going to construct manually most of the time so I'd say it makes sense in this case.